remix_01.py

```diff
@@ -8,7 +8,7 @@ def main():
     with open(filename, "r") as infile:
         remixes = json.load(infile)
         
-    for remix in remixes:
+    for remix in remixes[95:100]:
```

Comment:
I'm just loading a _sliced_ list to limit the data to just the json that gives this `UnicodeDecodeError` error.